pom.xml

```diff
@@ -2,26 +2,75 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
                       http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
-  <parent>
-    <groupId>org.neo4j.build</groupId>
-    <artifactId>parent-central</artifactId>
-    <version>46</version>
-    <relativePath/>
-  </parent>
 
   <groupId>org.neo4j</groupId>
   <artifactId>parent</artifactId>
   <version>3.4.0-SNAPSHOT</version>
-
   <packaging>pom</packaging>
   <name>Neo4j</name>
   <description>Neo4j Graph Database</description>
+  <inceptionYear>2002</inceptionYear>
+
+  <organization>
+    <name>The Neo4j Graph Database Project</name>
```

Comment:
Wow, really? I guess this was copied from a parent, but we may want to change this @magnusvejlstrup

Reply:
It's only used when generating copyright information in our javadoc since we don't use the site maven target anymore, i.e. `Copyright © 2002–2017 The Neo4j Graph Database Project. All rights reserved.`. Should maybe be changed to Neo4j Inc. now?